cypress/integration/login.spec.js

```diff
@@ -1,94 +1,83 @@
 import login from "../pages/loginModal.json";
 import headerLocators from "../pages/header.json";
 import organizationLocators from "../pages/organization.json";
+import Login from "../support/classes/login/index.js";
+
+const login1 = new Login();
 
 describe("Login", () => {
+  beforeEach(() => {
+    login1.setupTests();
```

Comment:
Why call it `login1`? I don't have anything against it, but it looks a bit confusing.